CudaArchitectureSelector.cmake

```diff
@@ -230,14 +238,15 @@ function(cas_get_architectures_by_name name output)
     set( kepler_version 3)
```

Comment:
Would it be possible to create a list of all architectures `set(cuda_arch_list "kepler;maxwell;pascal;volta;turing;ampere")` and then create these `arch_version` variables from this list ? 

We would then have to only change the list when a new arch needs to be added.

Reply:
I don't think we can do it that way? We need to use regex to match the versions as NVIDIA doesn't have a bulletproof system (every new architecture isn't a +10 for the first number, as we can see with Turing).

Reply:
Yes, you are right.